### ISSUE TITLE
Proper display of nested dict in param.

### ISSUE THREAD

**Issue by lkhphuc** [replicate/keepsake#575]
![Screenshot 2021-03-17 at 12 51 28](https://user-images.githubusercontent.com/12573521/111470202-803ab880-871f-11eb-8b4e-da6e06ec6ee4.png)

Support proper display of nested params file since many application provided a nested dictionary of parameters.
For example:
```
Params
data._target_:             project.data.MNISTDataModule
data.batch_size:         32
model.base:                resnet18
model.pretrain:          true
```

**Comment by bfirsh** [comment]
Yes! A simple solution would be to just format the JSON with newlines and indenting. Another alternative would be to have a YAML-like value, which might be easier to read and see the structure of.
